### DISCUSSION TITLE
Config error with zsh in vs-code

### DISCUSSION THREAD

**Discussion by tzabbi** [JanDeDobbeleer/oh-my-posh#5692] [Troubleshoot] [answered]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### What happened?

I have an issue with vs-code and zsh. I got a config error when I use vs-code in combination with zsh:
![image](https://github.com/user-attachments/assets/4ac0bea3-b1d1-4c6a-ab2a-2fc0228ab1f6)
When I'm using vs-code with bash everything works fine:
![image](https://github.com/user-attachments/assets/505fea10-8ade-48b7-8490-32585cc5a99e)

I check the font like in other issue described but it is the correct one:
![image](https://github.com/user-attachments/assets/5b8bc765-d0a7-49d1-9f1e-c38a475ca1f2)
![image](https://github.com/user-attachments/assets/21135e7a-e7c0-4d8f-bec6-cf1d4296e1bb)
![image](https://github.com/user-attachments/assets/e31a1ab1-974a-4b01-abdd-2b7936a1fd63)

What I see is that the zsh in vs-code uses the wrong path to the config (see log output). If you need more log output just let me know. :)

### Theme

Custom theme.

### Log output

```Shell
ZSH log:

Version: 23.15.1

Shell: zsh (5.9)

Prompt:

 toza   ~  CONFIG ERROR  

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   1 ms
Path(true)                                -   0 ms
Status(true)                              -   0 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 3.052525ms

Cache path: /home/toza/.cache/oh-my-posh

Config path: /home/toza/work/<some directories>/.config/ohmyposh/config.toml


Bash log:

```bash
Version: 23.15.1

Shell: bash (5.1.16(1)-release)

Prompt:


  toza  ⎈ <kube context>  ~/work/<path>  <git-branch> ≡
❯ 

Segments:

ConsoleTitle(true)                        -   0 ms
Os(true)                                  -   0 ms
Session(true)                             -   1 ms
Kubectl(true)                             -   2 ms
Path(true)                                -   0 ms
Git(true)                                 -   4 ms
Kubectl(false)                            -   0 ms
Time(false)                               -   0 ms
Executiontime(false)                      -   0 ms
Text(true)                                -   0 ms

Run duration: 8.623854ms

Cache path: /home/toza/.cache/oh-my-posh

Config path: /home/toza/.config/ohmyposh/config.toml
```

**Comment by JanDeDobbeleer** [comment, marked as answer]
@tzabbi this isn't going to be a bug inside oh-my-posh but something in your `.zshrc` or `zsh` configuration that makes oh-my-posh resolve a different path. How do you initialize oh-my-posh in `bash` vs `zsh`?

**Comment by tzabbi** [comment]
Arrg, yeah, you are right. 

```bash
❯ cat .zshrc | grep oh-my-posh
eval "$(oh-my-posh init zsh --config .config/ohmyposh/config.toml)"
```

Sorry for the trouble my bad.

**Reply by JanDeDobbeleer** [reply]
Glad you found the issue!
